client/lib/interpolate-components/Makefile

```diff
@@ -1,7 +1,23 @@
 REPORTER ?= spec
-MOCHA ?= ../../../node_modules/.bin/mocha
+
+ifneq ("$(wildcard node_modules/.bin/)","")
+NODE_BIN ?= ./node_modules/.bin
+else
+NODE_BIN ?= ../../../node_modules/.bin
+endif
+
+BABEL ?= $(NODE_BIN)/babel
+MOCHA ?= $(NODE_BIN)/mocha
 
 test:
 	@NODE_ENV=test NODE_PATH=test:../../../client $(MOCHA) --compilers jsx:babel/register --reporter $(REPORTER)
 
+publish:
+	@npm install
+	@make test
+	$(BABEL) -sd lib/ src/
+	@npm publish
+	rm -rf lib
+	rm -rf node_modules
```

Comment:
I see. So, do we really need to `npm install` here? I think it's going to search for a `node_modules` folder up the tree, will found one in `wp-calypso` and use it right?